bandersnatch/point.go

```diff
@@ -104,17 +104,16 @@ func (p *PointAffine) SetBytes(buf []byte) (int, error) {
 
 // Reads an uncompressed affine point
 // Point is not guaranteed to be in the prime subgroup 
-func ReadUncompressedPoint(r io.Reader) *PointAffine {
-	var x = make([]byte, 32)
-	var y = make([]byte, 32)
-	n, err := r.Read(x)
+func ReadUncompressedPoint(r io.Reader) PointAffine {
+	var xy = make([]byte, 64)
```

Comment:
This affects readability a bit, but I'm guessing that because we Read alot, making one allocation showed significant performance gains?

Reply:
Correct, this was at the hot path of `go-verkle` benchmarks. 

Also, something relevant to say about this is that both `x` and `y` have the same "lifetime", so bundling them in the same slice can't have unwanted consequences regarding GC. 

Note also that I avoid the pointer return since that confuses the compiler in escaping analysis.